Documentation/gettingstarted/k8s-install-download-release.rst

```diff
@@ -1,12 +1,10 @@
-Download the Cilium release tarball and change to the kubernetes install directory:
-
-.. parsed-literal::
-
-    curl -LO |SCM_ARCHIVE_LINK|
-    tar xzvf |SCM_ARCHIVE_FILENAME|
-    cd |SCM_ARCHIVE_NAME|/install/kubernetes
-
 `Install Helm`_ to prepare generating the deployment artifacts based on the
 Helm templates.
 
 .. _Install Helm: https://helm.sh/docs/using_helm/#install-helm
+
+Setup helm repository:
+
+.. code:: bash
+
+    helm repo add cilium http://releases.cilium.io/charts
```

Comment:
The use of HTTP rather than HTTPS here makes me a bit nervous, but this will require additional work on the server side.

Reply:
You could enabled CloudFront for [custom domain](https://aws.amazon.com/cloudfront/custom-ssl-domains/), this will allow you to issue free SSL certificate via ACM.

Reply:
Thanks for the pointers, I will discuss this with the domain administrator.